### ISSUE TITLE
Exclusions syntax

### ISSUE THREAD

**Issue by ghost** [claustromaniac/poop#24]
You're saying it accepts regex if seperated by :: but saving the options then reopening them removes a :  ... is this intended?

**Comment by ghost** [comment]
Can you confirm this is just a visual bug and regex is fine to use?

**Comment by claustromaniac** [comment]
Yes, sorry. Regex rules work, but this is not just a visual bug. Every time you change options, before saving you have to add the missing `:` for the rules to work as intended. I've been pretty busy but I intend to fix this and other issues soon.
